src/tribol/mesh/MfemData.cpp

```diff
@@ -901,9 +1140,15 @@ MfemJacobianData::UpdateData::UpdateData( const MfemMeshData& parent_data, const
   }
   // create a matrix transfer operator for moving data from redecomp to the
```

Comment:
Add comments as to what the transfer operators are. What 00, 01 and 10 mean.

Reply:
added to header